test/test_jit.py

```diff
@@ -7287,6 +7287,13 @@ def fn(x):
         with self.assertRaisesRegex(RuntimeError, "operation failed in interpreter"):
             fn(torch.tensor(4))
 
+    def test_trace_contiguous(self):
+        def foo(x):
+            return x[0].contiguous().view(3, 4)
```

Comment:
I'm wondering if a better test (than the expect test) might be to check if the output of foo shares the same storage with the input of foo